generator/views/glean_ping_view.py

```diff
@@ -294,6 +295,24 @@ def _make_dimension(
             ],
         }
 
+        if lookml["type"] == "time":
+            # Remove any _{type} suffix from the dimension group name because each timeframe
+            # will add a _{type} suffix to its individual dimension name.
+            lookml["name"] = re.sub("_(date|time(stamp)?)$", "", looker_name)
+            lookml["timeframes"] = [
+                "raw",
+                "time",
+                "date",
+                "week",
+                "month",
+                "quarter",
+                "year",
+            ]
+            # Dimension groups should not be nested (see issue #82).
+            lookml["label"] = f"{group_label}: {group_item_label}"
```

Comment:
Is this materially different than the previous label? `{group_label} {group_item_label}`

Reply:
I was just following the [precedent from `lookml_utils.py`](https://github.com/mozilla/lookml-generator/blob/7a386b47097030f7876361e45ce374314053a93f/generator/views/lookml_utils.py#L83), but I suppose it's slightly better to be more internally consistent with other metrics here than with that alternate implementation.  I'll remove this.